en/research-summaries/conversation-design.md

```diff
@@ -0,0 +1,617 @@
+<p> “Participant 4. Trying to find different numbers.” <p>
+<br>
+<p>Screen shows this text: </p>
+<p> “Because he uses the word "number", the agent matches it to the "Business number registration" task.” </p>
+<br>
+<p><b>Participant:</b></p>
+“Can you please call the CRA business enquiry number?”</p>
+<br>
+<p><b>Agent: </b></p>
+<p>“To get a business number, go to Business registration...” </p>
+<br>
+<p><b>Participant:</b></p>
+<p>“Can you give me the number of the Canada child benefit number?” </p>
+<br>
+<p><b>Agent: </b></p>
+<p> “To get a business number, go to Business registration...”</p>
+<br>
+<p><b>Participant:</b></p>
+<p> “Can you give me the CRA business number that is related to mistakes and errors on payroll?” </p>
+<br>
+<p><b>Agent:</b></p>
+<p> “To get a business number, go to...” </p>
+<br>
+<p>Screen shows this text: </p>
+<p> “The training phrases were chosen assuming users would listen to the welcome message and understand that they didn't need to say "give me the number for...". </p>
+</details>
+      </figcaption>  
+</figure>
+
+{:/}
+
+
+
+##### Vague questions are hard to answer
+
+When people speak to a conversational agent, some are super specific, while others are very vague. The agent needs to be able to handle the whole spectrum. 
+
+Compare these:
+
+- “I want to make a payment arrangement for my payroll taxes”
+- “I need help with my account”
+
+In the first case, the agent can match this to a specific intent: payment arrangements. It also has all it needs to give the proper answer because it knows the type of debt is payroll taxes. 
+
+In the second example, the agent cannot yet match the request with the payment arrangement “intent.” It must probe for more detail before knowing what to reply: “What do you want to do about your account?”
+
+In this project, we also realized that words can have multiple meanings:
+
+- my account - is the person talking about their bank account, or their CRA “My Account”?
+- payment - is the person talking about a payment to the CRA, or a payment from the CRA? 
+- number - is it about a phone number, or an account number?
+
+As a designer, you need to plan for vague requests, and design “intents” to handle them. 
+
+#### **When the conversation goes wrong**
+
+People expect conversational agents to understand them as well as a human would. When the conversation doesn’t go as planned, users get frustrated if the agent isn’t able to fix it elegantly. 
+
+People don’t like to repeat themselves.
+
+“Intents” have to be designed in a way that allows users to give as little or as much information as they want. When the user doesn’t give much, the agent needs to ask questions to fill in the gaps. When the user says a lot, the agent needs to retain these values in context and not ask for them again. 
+
+{::nomarkdown}
+
+<figure class="video_container">
+  <video controls="true" allowfullscreen="true">
+    <source src="../media/Clip3-EN.mp4" type="video/mp4">

```

Comment:
For some strange reason this video is so big (1080 instead of 720). I'll try to figure out a way to adjust it.